src/jgikbase/idmapping/storage/exceptions.py

```diff
@@ -1,11 +1,11 @@
-'''
+"""
 Exception classes thrown by the storage system.
 
 @author: gaprice@lbl.gov
```

Comment:
IMO author annotations are very unnecessary in the age of git. Not adamant on this point or anything.

Reply:
I've been removing them as I update the files to actually do something. PyDev sticks them in.